src/app.module.ts

```
@@ -22,6 +25,11 @@ import { MyLoggerModule } from './my-logger/my-logger.module';
         database: configService.dbName,
         autoLoadEntities: true,
         synchronize: false,
+        namingStrategy: new SnakeNamingStrategy(),
+
+        // Pour les environnements de développement (TypeScript)
+        entities: [__dirname + '/entities/**/*.entity{.ts,.js}'],
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)